charts/argo-templates/scripts/diff-bq-table.sh

```diff
@@ -15,6 +15,12 @@ for c in ${COMPARE_COLS//,/ }; do
 done
 declare -r FULL_DIFF=$(join_by ' OR ' "${COMPARISONS[@]}")
 
+declare -a PRIMARYCOLUMNS=()
+for col in ${PK_COLS//,/ }; do
+  PRIMARYCOLUMNS+=("${col} as datarepo_${col}")
+done
+declare -r SELECTCOLUMNS=$(join_by ', ' "${PRIMARYCOLUMNS[@]}")
```

Comment:
Similarly here:
```suggestion
declare -r SELECT_COLUMNS=$(join_by ', ' "${PRIMARYCOLUMNS[@]}")
```
I'm also not sure about "SELECT" as the descriptor for the columns, since we're selecting other things, too. How does `REPO_KEYS` sound as an alternative? Looking for something to highlight the fact that we're pulling out the primary-key values from the Jade dataset.

Reply:
REPO_KEYS makes more sense! I'll commit that.